docs/src/site/markdown/release-notes-2-1-0.md

```diff
@@ -0,0 +1,164 @@
+    </tr>
+    <tr>
+      <td>Support ORDER BY clause in GROUP_CONCAT</td>
+      <td>[TRAFODION-2270](https://issues.apache.org/jira/browse/TRAFODION-2270)</td>
+    </tr>
+    <tr>
+      <td>Support for boolean datatype</td>
+      <td>[TRAFODION-2099](https://issues.apache.org/jira/browse/TRAFODION-2099)</td>
+    </tr>
+    <tr>
+      <td>Added encryption functions for Trafodion</td>
+      <td>[TRAFODION-2228](https://issues.apache.org/jira/browse/TRAFODION-2228)</td>
+    </tr>
+    <tr>
+      <td>Support of SQL extension 'EXCEPT'</td>
+      <td>[TRAFODION-2117](https://issues.apache.org/jira/browse/TRAFODION-2117)</td>
+    </tr>
+    <tr>
+      <td>Support for tinyint,largeint,unsinged datatype in SQL</td>
+      <td>[TRAFODION-2086](https://issues.apache.org/jira/browse/TRAFODION-2086)</td>
+    </tr>
+    <tr>
+      <td>SQL syntax to support INTERSECT</td>
+      <td>[TRAFODION-2047](https://issues.apache.org/jira/browse/TRAFODION-2047)</td>
+    </tr>
+    <tr>
+      <td>Enable support for various non-ansi sql syntax/functionality</td>
+      <td>[TRAFODION-2180](https://issues.apache.org/jira/browse/TRAFODION-2180)</td>
+    </tr>
+    <tr>
+      <td>Use of CQD to set scratch directory locations</td>
+      <td>[TRAFODION2146](https://issues.apache.org/jira/browse/TRAFODION2146)</td>
+    </tr>
+  </table>
+</span>
+
+## Improvements
+
+<span>
+  <table>
+    <tr>
+      <th>Feature</th>
+      <th>Jira ID</th>
+    </tr>
+    <tr>
+      <td>Improve query compilation time</td>
+      <td>[TRAFODION-2137](https://issues.apache.org/jira/browse/TRAFODION-2137)</td>
+    </tr>
+    <tr>
+      <td>Improve upsert performance for table with indexes</td>
+      <td>[TRAFODION-1546](https://issues.apache.org/jira/browse/TRAFODION-1546)</td>
+    </tr>
+    <tr>
+      <td>Improve concurrency during DDL operations</td>
+      <td>[TRAFODION-2037](https://issues.apache.org/jira/browse/TRAFODION-2037)</td>
+    </tr>
+  </table>
+</span>
+
+## Install and Configuration Changes
+
+* Ambari Integration ([TRAFODION-2291](https://issues.apache.org/jira/browse/TRAFODION-2291))<br/>
+Integration with Ambari cluster manager ambari interface/integration. The Ambari integration provides support for Hortonworks Hadoop distributions, while the command-line Trafodion Installer supports Cloudera and Hortonworks Hadoop distributions, and for select vanilla Hadoop installations.
+* Python Installer ([TRAFODION-1839](https://issues.apache.org/jira/browse/TRAFODION-1839))<br/>
+Trafodion Installer Evolution The command-line installer has been replaced for the 2.1.0 release. Written in python, it replaces the legacy bash-script installer. The bash command-line installer is deprecated as of 2.1.0, but is still provided, just in case you experience any problems with the new installer. If so, please report those problems to the project team, since the legacy installer will soon be obsoleted.
```

Comment:
"will soon be obsoleted" should be "will soon be obsolete". Also, you might want to place a period after "Trafodion Installer Evolution".